src/plugins/operators/base.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from gobcore.message_broker.initialise_queues import initialize_message_broker
+from gobcore.message_broker.config import CONNECTION_PARAMS, WORKFLOW_EXCHANGE
+from gobcore.message_broker.message_broker import Connection
+
+log = logging.getLogger(__name__)
+
+
+class BaseMessageManager:
+    def consume_message(self, context, queue, key, message_id, ack=False):
+
+        def message_matched(msg):
+            header = msg["header"]
+            if header["dag_id"] == context['dag_run'].dag_id \
+                and header["run_id"] == context['dag_run'].run_id \
+                and header["message_id"] == message_id:
+                return True
```

Comment:
return header["dag_id"] == context['dag_run'].dag_id \
                and header["run_id"] == context['dag_run'].run_id \
                and header["message_id"] == message_id